README.md

```diff
@@ -20,6 +22,26 @@ This command will build and publish
 IMAGE_REGISTRY=example.com IMAGE_REPO=my-repo IMAGE_TAG=v1 bazel run //cmd/gcp-controller-manager:publish
 ```
 
+## Publishing cloud-controller-maanger image
+
+This command will build and publish cloud-controller-manager
+`gcr.io/k8s-image-staging/cloud-controller-manager:latest`:
```

Comment:
in k8s org we dont publish/promote latest tag, only tagged images

cc @sdmodi @aojea

Reply:
this is an example,

Reply:
i know, but people tend to use that 😅